Ollivanders/src/net/pottercraft/Ollivanders2/Ollivanders2.java

```diff
@@ -2017,6 +2093,28 @@ private boolean listAllIngredients (Player player)
       return true;
    }
 
+   /**
+    * Lists all the potions active in the game.
+    *
+    * @param player the player to display the list to
+    * @return true
+    */
+   private boolean listAllPotions (Player player)
+   {
+      String displayString = "Potions:";
```

Comment:
probably a little better as a StringBuffer.

Reply:
Yep, there are lots of places we need to fix - https://github.com/Azami7/Ollivanders2/issues/227